.vscode/settings.json

```diff
@@ -71,13 +67,15 @@
     "dequips",
     "dim",
     "dim's",
+    "Dont",
```

Comment:
Is this correct?

Reply:
DontForgetDupes

Reply:
I figured there was a good reason for them. Thanks!